src/server.js

```diff
@@ -19,6 +21,14 @@ export const startServer = port => new Promise((resolve, reject) => {
     reject(new Error('The server must be started with an available port'));
   }
 
+  passport.serializeUser((user, cb) => {
```

Comment:
@ssanusi Can shorten the function, as you are just returning the callback function